bower.json

```diff
@@ -26,10 +25,9 @@
   ],
   "license": "MIT",
   "private": false,
-  "devDependencies": {
+  "dependencies": {
```

Comment:
we should split this so that jquery and select2 are still under devDependencies and bootstrap is under dependencies as we are currently bundling the src files for jquery and selec2 in with our techne.min.js dist file